code/core/src/cli/helpers.ts

```diff
@@ -164,7 +163,7 @@ export async function copyTemplateFiles({
   renderer,
   language,
   destination,
-  includeCommonAssets = true,
+  commonAssetsDir,
```

Comment:
injecting common assets from `create-storybook` into this shared util file

Reply:
What common assets? Can you explain this change a bit more?

Reply:
I think it is correct. Those assets are installed by `create-storybook` into the user directory. So it makes sense that it is part of `create-storybook`. require.resolve('create-storybook') gave my build errors, as this shared util file doesn't know about `create-storybook`. So therefore I injected the asset dir.

I also tried moving this function altogether to `create-storybook` but it gave me too much unit test and build errors, that is why I not able to resolve in time.